CharlieBackend.Api/Controllers/StudentsController.cs

```diff
@@ -54,7 +54,7 @@ public async Task<ActionResult> PostStudent(long accountId)
         /// Get student information by student id
         /// </summary>
         /// <response code="200">Successful return of student</response>
-        /// <response code="409">Error, can not find student</response>
+        /// <response code="404">Error, can not find student</response>
         [SwaggerResponse(200, type: typeof(StudentMock))]
```

Comment:
```suggestion
        [SwaggerResponse(200, type: typeof(StudentDto))]
```
We need to either update it to `StudentDto`, or rename `StudentMock` to something else, so we won't confuse our API users:

![Screenshot_2021-01-26 Swagger UI](https://user-images.githubusercontent.com/38729881/105842646-2f3ffb00-5fdf-11eb-8149-1ef8b7dac02b.png)

I think the first option is preferable here.

Reply:
Okay, I'll fix that. But I cannot answer the question why here StudentMock, because it was written before I took this task.

Reply:
Okay, I'll fix that. But I cannot answer the question why here StudentMock, because it was written before I took this task.